sortfilterproxymodel.qbs

```diff
@@ -31,6 +31,7 @@ Product
     }
 
     cpp.includePaths: [ "." ]
+    cpp.cppFlags: "-std=c++11"
```

Comment:
@MarcelBlanck please see [this answer](https://github.com/Proemion/SortFilterProxyModel/pull/5#issuecomment-319290242)